README.md

```diff
@@ -54,18 +35,18 @@ or starting a container manually:
 $ docker run --detach \
 		--name thelounge \
 		--publish 9000:9000 \
-		--volume ~/.thelounge:/home/lounge/data \
+		--volume ~/.thelounge:/var/opt/thelounge \
 		--restart always \
-		thelounge/lounge:latest
+		thelounge/thelounge:3.0.0-pre.7
```

Comment:
This should remain `latest`?

Reply:
yep! but latest doesn't exist on the new repo yet, will be created when 3.0.0 is released

Reply:
Well it's either that, or hardcoded outdated version.

Reply:
My idea was to keep the `master` documentation refer to the latest stable release (2.7.1) and have this branch be sort of wip until 3.0.0 is released. There's also a [link](https://github.com/thelounge/thelounge-docker#v3-pre-releases) to the v3 docs in the 2.7.1 readme file